src/main/java/org/opentripplanner/model/plan/Emissions.java

```diff
@@ -0,0 +1,18 @@
+package org.opentripplanner.model.plan;
+
+import org.opentripplanner.framework.lang.Sandbox;
+import org.opentripplanner.framework.model.Grams;
+
+@Sandbox
+public class Emissions {
+
+  private Grams co2;
+
+  public Grams getCo2() {
+    return co2;
+  }
+
+  public void setCo2(Grams co2) {
```

Comment:
Please make this an immutable class where you have to pass everything in the constructor and you check for null.